.mergify.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+# Format Ref: https://doc.mergify.io/configuration.html
+pull_request_rules:
+    - name: 'After Cirrus-CI Fails'
+      conditions:
+          - '- status-failure ~= cirrus-ci/.*'
```

Comment:
This says, match any pull request where the status name matches a regular-expression.  In this case, `cirrus-ci/.*` would  match all tasks with that prefix.  In this PR there is only one: https://github.com/containers/libpod/pull/5392/files#diff-0f7067c31caedf7df2739c9b07efa122R89 for testing.